client/src/Components/Activities/Activities.js

```diff
@@ -36,4 +37,14 @@ const Activities = ({ activities, addEntry, setFeeling }) => {
   )
 }
 
-export default Activities;
\ No newline at end of file
+export default Activities;
+
+Activities.propTypes = {
+  activities: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
Oooo the `.shape()` is a new one for me! Great call implementing this. Definitely want to keep this method in mind when using this package in the future.

Reply:
Shape is a new one for me too! This looks great, super thorough.